files/www/cgi-bin/advancedconfig

```diff
@@ -175,7 +175,7 @@ local settings = {
         category = "WAN Settings",
         key = "aredn.wan.vlanid",
         type = "string",
-        desc = "Specify WAN VLAN #",
+        desc = "Specify WAN VLAN number; must be an integer in the range [10,4096]",
```

Comment:
Is it 10-4096 or 1-4096?

Reply:
It should be 1 to 4094.   The use case is to work around  switch constraints, and avoid to impose further constraints.   The fact that some switch vendors use vlan '1' for management purposes is a reason to use another ID.    

Note. the range is 0 to 4095 or 4096 total values.  0 and 4095 are reserved in the spec.   0 is used to give packet priority through a switch and considered untagged.   Don't recall what 4095 or 0xFFF is reserved for.

The constraint AREDN would have -- check what vlan dtdlink is in use, and prevent that from being a selection.  Or give an error if it is selected with the conflict.

Reply:
Thanks, guys.  Change is made in latest commit.